<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>677</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Stabilizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="outerSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="pageSelectorSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="pageSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Connections</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Convergence</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Instruments</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Settings</string>
         </property>
        </item>
       </widget>
       <widget class="QStackedWidget" name="pages">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="connectionsPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="connectionsLabel">
            <property name="text">
             <string>Currently detected connections</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="connectionsText">
            <property name="font">
             <font>
              <family>Lucida Console</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="convergencePage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="worldStateLabel">
            <property name="text">
             <string>World state</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="worldStateText">
            <property name="font">
             <font>
              <family>Lucida Console</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="convergedStateLabel">
            <property name="text">
             <string>Converged state</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="convergedStateText">
            <property name="font">
             <font>
              <family>Lucida Console</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="instrumentsPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="instrumentsLabel">
            <property name="text">
             <string>Currently connected instruments</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="instrumentsText">
            <property name="font">
             <font>
              <family>Lucida Console</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settingsPage">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="ipAddressLabel">
            <property name="text">
             <string>IP Address</string>
            </property>
            <property name="buddy">
             <cstring>ipAddress</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ipAddress">
            <property name="text">
             <string>127.0.0.1</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="pluginsLabel">
            <property name="text">
             <string>Loaded plugins</string>
            </property>
            <property name="buddy">
             <cstring>plugins</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListView" name="plugins">
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="lowerSection">
        <item>
         <widget class="QLabel" name="logLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Log</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="logView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="footer">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="startOrStopListeningButton">
            <property name="text">
             <string>Start Listening</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>logView</zorder>
       <zorder>logLabel</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_log"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_log">
   <property name="text">
    <string>Save log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pageSelector</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>pages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
